package.json

```diff
@@ -51,7 +51,8 @@
     "peer-info": "^0.7.1",
     "pre-commit": "^1.1.3",
     "run-parallel": "^1.1.6",
-    "run-series": "^1.1.4"
+    "run-series": "^1.1.4",
+    "webpack": "2.1.0-beta.25"
```

Comment:
There was a breaking change in a webpack beta which cause builds to fail, I pushed a new release fixing it out yesterday, so this should not be needed, only an aegir upgrade.